sahha-android/src/main/java/sdk/sahha/android/data/repository/SensorRepoImpl.kt

```diff
@@ -708,7 +710,7 @@ internal class SensorRepoImpl @Inject constructor(
     }
 
     private suspend fun clearLocalPhoneScreenLockData() {
-        deviceDao.clearUsages()
+//        deviceDao.clearAllUsages()
```

Comment:
This method should either be removed or uncommented. What do you think?

Reply:
yep, removed some functions that weren't being used, including that one